frontend/src/components/NavMenu.vue

```diff
@@ -11,10 +11,10 @@
     <v-list nav>
       <v-list-item to="/">Home</v-list-item>
       <v-list-item to="/about">About</v-list-item>
-      <v-list-item v-if="appBarDisplay" :to="`/info/${defaultGame}`">Game</v-list-item>
+      <v-list-item @click="gameSelected = !gameSelected"> Games </v-list-item>
+      <v-list-item v-if="gameSelected" :to="`/info/${defaultGame}`">Game</v-list-item>
```

Comment:
Add a v-divider before the game to add some separation between the games selected.